### PR TITLE
Lithuanian, Polish, Russian, Spanish, Ukrainian

### PR DESCRIPTION
**Pull request by 1ec5**
Pulled the latest translations from Transifex and added new languages to the iOS and macOS projects. Added .stringsdict files according to issues posted to Transifex by translators. The tx configuration included languages with at least one file translated 80% or more. Lithuanian, Polish, Russian, Spanish, and Ukrainian are all partial translations at this time.

/cc @Andygol

### CODE REVIEW

**Comment by vkurchatkin** [comment]
@1ec5 not sure if you are interested in such details, but Russian nouns can have up 3 plural forms and requires some logic to choose the right one. Not a big deal, but can be a bit cringeworthy when it's wrong.

**Comment by 1ec5** [comment]
Thanks @vkurchatkin – can you point out what the singular/dual/plural forms should be?

**Comment by vkurchatkin** [comment]
According to this: http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#ru you have to specify `one`, `many` and `few`.

**Comment by 1ec5** [comment]
Can you fill out the following table?

English | `one` | `many` | `few`
----|----|----|----
degree
minute
second

Thanks!

**Comment by vkurchatkin** [comment]
Sure:

|  English  | `one` | `few` | `many`
| -------- | ------- | ----- | -------
|   degree | градус | градуса | градусов
|   minute | минута | минуты | минут
|   second | секунда | секунды | секунд

One caveat: this is only nominative case, so these forms are not applicable in some contexts.

**Comment by 1ec5** [comment]
Awesome – fixed in e4eca526e97a7a82cdef27953f859370a964d753.

**Comment by vkurchatkin** [comment]
It seems that Polish and Ukrainian are similar to Russian in this regard

File: platform/darwin/resources/pl.lproj/Foundation.stringsdict [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		<key>minutes</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>d</string>
+			<key>one</key>
+			<string>%d minuta</string>
+			<key>other</key>
+			<string>%d minuty</string>
```
Comment (1ec5) [inline platform/darwin/resources/pl.lproj/Foundation.stringsdict]: @codecalm, can you fill out this table?

English | `one` | `few` | `many`
----|----|----|----
degree
minute
second

**Comment by 1ec5** [comment]
The relevant messages haven’t been translated into Ukrainian on Transifex, but perhaps @Andygol or @mourner could help with the plural forms for “degree”, “minute”, and “second” too.

> this is only nominative case, so these forms are not applicable in some contexts.

Unfortunately, this is generally the case with NSFormatter subclasses on Apple platforms. If an application needs a different case, it normally needs to come up with its own localizable strings to be translated in context.

File: platform/darwin/resources/pl.lproj/Foundation.stringsdict (hunk 1 above)
Comment (codecalm) [inline platform/darwin/resources/pl.lproj/Foundation.stringsdict]: Sure

English | `one` | `few` | `many`
--|--|--|--
degree | stopień | stopnie | stopni
minute | minuta | minuty | minut
second | sekunda | sekundy | sekund

Comment (1ec5) [inline platform/darwin/resources/pl.lproj/Foundation.stringsdict]: Thanks – incorporated in f76cfe5c0603af076fced8c10053c5f0f9b90a8d.

**Comment by Andygol** [comment]
>The relevant messages haven’t been translated into Ukrainian

Here is Ukranian translation 🇺🇦 

|  English  | `one` | `few` | `many`
| -------- | ------- | ----- | -------
|   degree | градус | градуси | градусів
|   minute | хвилина | хвилини | хвилин
|   second | секунда | секунди | секунд

/cc @1ec5 @mourner

**Pull request merged**